kernel/src/lib.rs

```diff
@@ -86,7 +86,7 @@ pub fn main() {
     let mut affinity = CpuSet::new_empty();
     affinity.add(CpuId::bsp());
     ThreadOptions::new(init_thread)
-        .priority(Priority::idle())
+        .priority(Priority::IDLE)
```

Comment:
If we officially switch to the class-based scheduler, then the thread option should take `SchedPolicy`, not `Priority`.